webpack/src/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './PluginName';
```

Comment:
```suggestion
export { default } from './ForemanPluginName';
```
Also the file needs renaming :)